wp-content/mu-plugins/pantheon/pantheon-updates.php

```diff
@@ -15,12 +16,54 @@ function _pantheon_hide_update_nag() {
 	remove_action( 'network_admin_notices', 'update_nag', 3 );
 }
 
-// Compare the current WordPress version to the latest available
+// Get the latest WordPress version
+function _pantheon_get_latest_wordpress_version() {
+	$core_updates = get_core_updates( array('dismissed' => false) );
+
+	if( ! is_array($core_updates) || empty($core_updates) || ! property_exists($core_updates[0], 'current' ) ){
+		return null;
+	}
+
+	return $core_updates[0]->current;
+}
+
+// Check if WordPress core is at the latest version.
+function _pantheon_is_wordpress_core_latest() {
+	$latest_wp_version = _pantheon_get_latest_wordpress_version();
+
+	if( null === $latest_wp_version ){
+		return true;
+	}
+
+	// include an unmodified $wp_version
+	include( ABSPATH . WPINC . '/version.php' );
```

Comment:
What context is this code being called from? It seems there should be an official way to get the current running WordPress version. Is there some reason we need to include version.php here?

Reply:
most official way would be `global $wp_version`, I think the include is to ensure that $wp_version hasn't been otherwise modified on the way here.